test/unit/event-handler-wrap-error-handler-test.ts

```diff
@@ -1,17 +1,16 @@
 import simple from "simple-mock";
-import { test } from "tap";
-import { wrapErrorHandler } from "../../pkg/dist-src/event-handler/wrap-error-handler";
+import { wrapErrorHandler } from "../../src/event-handler/wrap-error-handler";
 
 test("error thrown in error handler", (t) => {
   t.plan(2);
 
   const messages = [];
   simple.mock(console, "log", messages.push.bind(messages));
-  t.doesNotThrow(() => {
+  expect(
     wrapErrorHandler(() => {
       throw new Error("oopsydoopsy");
-    }, new Error("oops"));
-  });
+    }, new Error("oops"))
+  ).not.toThrow();
 
   t.ok(messages.find((message) => /FATAL/.test(message)));
```

Comment:
Not sure what to do with these either

Reply:
maybe use [`.toBeTruthy`](https://jestjs.io/docs/en/expect#tobetruthy)?

```suggestion
  expect(messages.find((message) => /FATAL/.test(message))).toBeTruthy();
```